server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/PermissionSourceType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.core.shared.constant;
+
+/**
+ * @author gaoda.xy
+ * @date 2024/1/2 19:45
+ */
+public enum PermissionSourceType {
```

Comment:
concept 'source' seems a little hard to understand.

for all permissions, it shoud be an 'authorization' domain problem, how about AuthorizationType ?